sys/test.txt

```diff
@@ -187,3 +187,12 @@ syz_end_var_struct {
 	f1	const[0x42, int32be]
 	f2	flags[syz_end_flags, int64be]
 } [packed]
+
+# Const buffer type
+
+syz_test$bufconst(a0 ptr[in, syz_bufconst])
+
+syz_bufconst {
+	f0	bufconst["wlan0"]
```

Comment:
a simple way is use zero array to pad the name, such as:
name bufconst["wlan0"]
namepad array[const[0, int8], 58]   // the buf size is 64, the bufconst size is 6.  64-6 = 58.